src/components/graph-view.js

```diff
@@ -1302,6 +1303,8 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
 
     const t = d3.zoomIdentity.translate(x, y).scale(k);
 
+    this.props.onZoomCall({ k, x, y });
```

Comment:
I'd be curious to hear people's thoughts on this, originally I was going to put it in the `handleZoom` function, but that gets triggered several times as the zoom progresses, which is a problem for the flow because it would cause the `diagram-builder` to re-render more times than was necessary for every zoom call. 

This only gets called to initialize the zoom and should theoretically be accurate information for what the next viewTransform will be. We could also still use `onZoomEnd` to ensure that the values are kept in sync and update appropriately in the flow view if isn't.

Reply:
from the name it is not clear to me when exactly its get called or purpose of the prop, also adding the comment might help

Reply:
Yup, Mikhail mentioned changing the new to `onZoomStart` above, so it should be more clear now when it's called